tsl/src/nodes/decompress_chunk/decompress_chunk.c

```diff
@@ -338,7 +338,7 @@ smoothstep(double x, double start, double end)
 {
 	x = (x - start) / (end - start);
 	x = x < 0 ? 0 : x > 1 ? 1 : x;
-	return x * x * (3.0f - 2.0f * x);
+	return x * x * (3.0F - 2.0F * x);
```

Comment:
I find it odd that the recommendation is to use uppercase constant suffixes, but if this is the default, then we should use it.